Package/FuzzyMatcher.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace OpenTap.Package
+{
+    internal class FuzzyMatcher
```

Comment:
I wanted more fine grained control over our matching instead of using some stock matcher. I would also prefer to avoid adding dependencies since the algorithm is pretty simple anyway